docs/design/features/typemap.md

```diff
@@ -188,14 +188,12 @@ An entry in an External Type Map is included when the "trim target" type is refe
 - The argument to the `isinst` IL instruction.
 - The argument to the `castclass` IL instruction.
 - The argument to the `box` instruction.
+  - If the trimming tool can determine that this box does not escape and could be stack allocated, it can ignore this `box` instruction and any corresponding `unbox` or `unbox.any` instructions.
 - The argument to the `mkrefany` instruction.
 - The argument to the `refanyval` instruction.
 - The argument to the `newarr` instruction.
-- The argument to the `ldobj` instruction.
-- The argument to the `stobj` instruction.
-- The argument to the `.constrained` instruction prefix.
-- The type of a method argument to the `newobj` instruction.
-- The owning type of the method argument to `call`, `callvirt`, `ldftn`, or `ldvirtftn`.
+- The type of a method argument to the `newobj` instruction if it is a class type.
+- The owning type of an instance method argument to `call` or `ldftn`, or the owning type of any method argument to `callvirt` or `ldvirtftn`.
```

Comment:
nit: Would it be worth it to make .il tests that exercise all these cases?

Reply:
I'll try to add some cases in IL.